dashboard/src/components/DeploymentFormBody/BasicDeploymentForm/SliderParam.tsx

```diff
@@ -101,7 +101,9 @@ class SliderParam extends React.Component<ISliderParamProps, ISliderParamState>
 
   private handleParamChange = (value: number) => {
     this.props.handleBasicFormParamChange(this.props.param)({
-      currentTarget: { value: `${value}${this.props.unit}` },
+      currentTarget: {
+        value: this.props.param.type === "string" ? `${value}${this.props.unit}` : value,
```

Comment:
the `unit` prop is now optional. Can you update the rest of the class to cover the case in which unit is not set? (lines 9 and 91)

Reply:
Please tell me if something went wrong when merging ... I'm not used to git, so I've spent more time learning about git than coding.